pkg/config/config.go

```diff
@@ -104,19 +105,22 @@ func (config *Config) Validate(cliMode bool) error {
 
 func validateMetrics(metricsCriteria Metric) error {
 	threshold := metricsCriteria.Threshold
+	if threshold < 0 {
```

Comment:
Why not move it inside case: ?

Reply:
That would mean checking for `threshold < 0` in 3 parts (more times as we support more metrics). Also, I can't think of a metrics that can have negative thresholds, which might be possible, but we haven't dealt with one yet.